CHANGELOG.md

```diff
@@ -32,10 +32,18 @@ Support for this is now added through the `add_computed_field` API.
 
 Read more about the session argument for computed fields in the [docs](https://hasura.io/docs/1.0/graphql/manual/api-reference/schema-metadata-api/computed-field.html).
 
+### CLI: Support servers with self-signed certificates (close #4564) (#4582)
+
+A new global flag is added to the CLI for working with GraphQL Engine servers with
```

Comment:
Cool. 

Minor: the first line and second line are both similar. If you read it, it seems little repeating. Can we combine then to be one line?